pypeit/core/findobj_skymask.py

```diff
@@ -6,10 +6,11 @@
 import copy
 
 import numpy as np
-import scipy
-from matplotlib import pyplot as plt
+import scipy.interpolate
+import scipy.ndimage
+import matplotlib.pyplot as plt
 
-from astropy import stats
+import astropy.stats
```

Comment:
Thanks.  I've found in my own work that including the full module path in the code is helpful so as to avoid scrolling back up to remember which package something came from.  I will leave these as-is, but will not be overly zealous in the future.